games/ninswitch/game_simple_reset.py

```diff
@@ -44,6 +66,12 @@ async def on_init(self):
     on_config, on_prepare, on_pre_game, on_countdown, on_start...
     """
 
+    async def on_prepare(self):
+        send_sig(nsg_reset, ON)
```

Comment:
I hope that resetting the board between games is enough, so I would propose removing `reset_trinkets` class altogether.

Then I would wrap lines 70-74 to a method, and just call it here:
```python
async def on_prepare(self):
    await self.reset_trinket()
```

Reply:
I tested it and what is proposed doesn't work directly. However this does work
```python
async def on_prepare(self):
        await ResetTrinkets().on()
        await ResetTrinkets().off()
        await asyncio.sleep(2)
```